### PR TITLE
Update diagrams and Managed Subscribe Readme

### PR DESCRIPTION
**Pull request by devkelley**
This PR does the following things:
- Updates Managed Subscribe sequence diagrams to show that the InVehicle Digital Twin and Managed Subscribe Module are under the same service of Ibeji.
- Adds simplified sequence diagrams from the perspective of a consumer and a provider to show what actually is needed from each of them.

### CODE REVIEW

File: docs/design/diagrams/managed_subscribe_interceptor_sequence.puml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 @startuml Managed Subscribe Interceptor
 title Managed Subscribe Interceptor Sequence
 participant "Digital Twin Provider" as DTProvider
+box "Ibeji"
 participant "Managed Subscribe Module" as ManagedSubscribeModule
```
Comment (ashbeitz) [inline docs/design/diagrams/managed_subscribe_interceptor_sequence.puml:5]: I feel that you should have abstraction parity amongst all of your actors inside of the Ibeji border.  You use module for Managed Subscribe and service for fundamental Ibeji functionality.

To fix this, could you rename "Managed Subscribe Module" to "Managed Subscribe Interceptor"?  Alternatively, you could rename "InVehicle Digital Twin" to "Fundamental Ibeji Module" (or something like that).

File: docs/design/diagrams/managed_subscribe_module_sequence.puml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 @startuml Managed Subscribe Module
 title Managed Subscribe Module Sequence
 participant "Digital Twin Provider" as DTProvider
-participant Ibeji
+box "Ibeji"
+participant "InVehicle Digital Twin" as Ibeji
 participant "Managed Subscribe Module" as ManagedSubscribeModule
```
Comment (ashbeitz) [inline docs/design/diagrams/managed_subscribe_module_sequence.puml:6]: Same concern as earlier - try to keep your abstraction parity the same.

**Review by ashbeitz — COMMENTED** [review]
A few comments.  Please take a look.

File: docs/design/diagrams/managed_subscribe_interceptor_sequence.puml (hunk 1 above)
Comment (ashbeitz) [reply docs/design/diagrams/managed_subscribe_interceptor_sequence.puml:5]: Devin and I chatted about this.  I'm fine with how it is today.  Devin will later provide a sequence diagram that shows what is happening inside the Managed Subscribe Module during this use case.

**Review by ashbeitz — APPROVED** [review]
Approved.

File: samples/managed_subscribe/.accepted_words.txt [hunk 3]
```diff
@@ -12,9 +12,11 @@ github
 http
 https
 Ibeji
+InVehicle
 invehicle
 MQTT
 repo
 svg
+TopicManagementCB
```
Comment (jorchiu) [inline samples/managed_subscribe/.accepted_words.txt:20]: What does `CB`  mean? Does it stand for callback?

Comment (devkelley) [reply samples/managed_subscribe/.accepted_words.txt:20]: Correct, it stands for callback

**Review by jorchiu — APPROVED** [review]

File: docs/design/diagrams/managed_subscribe_consumer_sequence.puml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+title Managed Subscribe Sequence from Consumer's Perspective
+participant "Digital Twin Consumer" as DTConsumer
+participant Ibeji
+participant Broker
+
+autonumber
+
+DTConsumer -> Ibeji: FindById - request
+DTConsumer <- Ibeji: FindById - response
+DTConsumer -> Ibeji: GetSubscriptionInfo (entity_id, constraints {freq, acc, etc})
```
[the remark this answers is not in the document]
Comment (devkelley) [reply docs/design/diagrams/managed_subscribe_consumer_sequence.puml]: removed content

File: docs/design/diagrams/managed_subscribe_provider_sequence.puml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    operations: ["ManagedSubscribe"]
+    uri: "Provider callback uri"
+    context: ""
+}
+end note
+Ibeji -> DTProvider: Register - response
+
+Ibeji -> DTProvider: TopicManagementCB (action, payload)
+note right of DTProvider
+// Tells provider to start publishing
```
[the remark this answers is not in the document]
Comment (devkelley) [reply docs/design/diagrams/managed_subscribe_provider_sequence.puml]: removed // for any comments that were not right next to a varaible

**Review by wilyle — APPROVED** [review]

**Pull request merged**